.tekton/python-tracer-prepuller.yaml

```
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: python-tracer-prepuller
+spec:
+  selector:
+    matchLabels:
+      name: python-tracer-prepuller
+  template:
+    metadata:
+      labels:
+        name: python-tracer-prepuller
+    spec:
+      # Configure an init container for each image you want to pull
+      initContainers:
+        - name: prepuller-git
+          # alpine/git:2.43.0
+          image: alpine/git@sha256:6ff4de047dcc8f0c7d75d2efff63fbc189e87d2f458305f2cc8f165ff83309cf
+          command: ["sh", "-c", "'true'"]
+        - name: prepuller-google-cloud-pubsub
+          # egymgmbh/pubsub-emulator:gh-mb117
+          image: egymgmbh/pubsub-emulator@sha256:88897fa72337b22a8edabf17a8b30bf9d9c6388b7c7e6d8c2b5e5c96d73fede1
+          command: ["sh", "-c", "'true'"]
+        - name: cassandra
```

Just for consistency in naming the `initContainers`.
```suggestion
        - name: prepuller-cassandra
```

---

Done